diff --git a/tests/phpunit/tests/fonts/font-face/wpPrintFontFacesFromStyleVariations.php b/tests/phpunit/tests/fonts/font-face/wpPrintFontFacesFromStyleVariations.php
--- a/tests/phpunit/tests/fonts/font-face/wpPrintFontFacesFromStyleVariations.php
+++ b/tests/phpunit/tests/fonts/font-face/wpPrintFontFacesFromStyleVariations.php
@@ -16,8 +16,8 @@ class Tests_Fonts_WpPrintFontFacesFromStyleVariations extends WP_Font_Face_UnitT
 	const FONTS_THEME = 'fonts-block-theme';
 
 	public static function set_up_before_class() {
+		self::$requires_switch_theme_fixtures = true;
 		parent::set_up_before_class();
-		self::$requires_switch_theme_fixtures = true;
 	}
 
 	/**
